crates/shared/src/trade_finding/zeroex.rs

```diff
@@ -58,7 +58,9 @@ impl Inner {
                 sell_amount,
                 buy_amount,
                 slippage_percentage: None,
+                taker_address: None,
                 excluded_sources: self.excluded_sources.clone(),
+                intent_on_filling: false,
```

Comment:
Wasn't this supposed to be `true` or rather behind a flag now?

Reply:
> The PR only changed this for the solvers, not price estimators. I will ask the 0x team if they think it would be a good idea to enable this also for price estimation

Makes sense. 👍 

> That being said, this would lead to slightly different prices between price estimation and solving, so maybe we should?

I think we don't necessarily need to have the same quoting properties for `/solve` and `/quote`.